ccsdspy/interface.py

```diff
@@ -99,7 +99,7 @@ def load(self, file):
 
         Returns
         -------
-        `OrderedDict` mapping field names to NumPy arrays.
+        sorted dictionary mapping field names to NumPy arrays.
```

Comment:
This states that the dictionary is sorted but I don't see any code that actually sorts the dictionary. For clarity I would also describe how it is sorted (alphabetically by key?)

Reply:
Good catch on being clear! I meant that the dictionary keys are ordered by the order the corresponding field appears in the packet definition (which is not necessarily alphabetical). Previously an OrderedDict was use to force the order, since older versions of dict didn't make guarantees on keeping insertion order.  Now,  the order of a dict's keys are the order they are inserted in (which is why you don't seen an explicit call to `sort()`).